swagger2.0/oic.wk.con.swagger.json

```diff
@@ -298,100 +265,41 @@
             }
           ]
         },
-
-        "n" :
-                {
+        "n": {
```

Comment:
add here in the description that it conveys the string to be set on the resource. 
this one is different than n in the get which can be a $ref